coral/types/utils.d.ts

```diff
@@ -1,17 +1,24 @@
 import type { operations, components } from "types/api.d";
 
-type KlawApiResponse<OperationId extends keyof operations> =
-  operations[OperationId]["responses"][200]["content"]["application/json"];
+type KlawApiResponse<OperationId extends keyof operations> = Prettify<
+  operations[OperationId]["responses"][200]["content"]["application/json"]
+>;
 type KlawApiModel<Schema extends keyof components["schemas"]> =
   components["schemas"][Schema];
 type KlawApiRequest<OperationId extends keyof operations> =
   operations[OperationId]["requestBody"]["content"]["application/json"];
 type KlawApiRequestQueryParameters<OperationId extends keyof operations> =
   operations[OperationId]["parameters"]["query"];
 
+type Prettify<T> = {
```

Comment:
I think something a bit more precise like `ResolveIntersectionTypes` would be better that `Prettify`

Reply:
Yes that makes sense! :D